subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -890,6 +890,10 @@ uint8_t ll_adv_enable(uint8_t enable)
 		memset(&conn_lll->conn_meta, 0, sizeof(conn_lll->conn_meta));
 #endif /* CONFIG_BT_CTLR_CONN_META */
 
+#if defined(CONFIG_BT_CTLR_LE_PING)
```

Comment:
nitpick. Move this block after `conn->procedure_expire = 0;` to be consistent with the order of assignments in ull_master.c